lightning/src/ln/channelmanager.rs

```diff
@@ -2487,14 +2487,15 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	fn internal_funding_created(&self, counterparty_node_id: &PublicKey, msg: &msgs::FundingCreated) -> Result<(), MsgHandleErrInternal> {
 		let ((funding_msg, monitor), mut chan) = {
 			let last_block_hash = *self.last_block_hash.read().unwrap();
+			let last_block_height = self.latest_block_height.load(Ordering::Acquire) as u32;
```

Comment:
It would probably be worth combining these behind a single lock and perhaps passing these around as a single type -- if there's a good name for it -- given they tend to be updated together. Let me see if I can come up with something reasonable. Suggestions welcome.

Reply:
Ended up combining these into a `BestBlock` struct behind a single lock. I can do the same in `ChannelMonitor` if it looks good.

Reply:
Sounds good to me.

Reply:
Updated `ChannelMonitor` accordingly.